packages/firestore/lite/src/api/snapshot.ts

```diff
@@ -82,23 +78,26 @@ export interface FirestoreDataConverter<T> {
   /**
    * Called by the Firestore SDK to convert a custom model object of type T
    * into a plain Javascript object (suitable for writing directly to the
-   * Firestore database). To use `set()` with `merge` and `mergeFields`,
-   * `toFirestore()` must be defined with `Partial<T>`.
+   * Firestore database). Used with {@link setData()}, {@link WriteBatch#set()}
+   * and {@link Transaction#set()}}.
    */
   toFirestore(modelObject: T): DocumentData;
+
+  /**
+   * Called by the Firestore SDK to convert a custom model object of type T
+   * into a plain Javascript object (suitable for writing directly to the
+   * Firestore database). Used with {@link setData()}, {@link WriteBatch#set()}
```

Comment:
I am hoping it looks nice in the final output, but I am not sure. A lot of the old code did this instead:

`{@link setDoc setDoc()}`

I hope this is the same as `{@link setDoc()}`.

Reply:
Oh -- I meant "(suitable for....).   Not a big deal though.

Reply:
Okay, got it. I do kind of prefer to leave the parenthesis as is. Otherwise, we have to add a conjunction of some sort which will make this part harder to parse.